src/Pages/FormPage/FormPage.tsx

```diff
@@ -160,34 +84,29 @@ const FormPage = () => {
           "Access-Control-Allow-Headers":
             "Origin, Content-Type, X-Amz-Date, Authorization, X-Api-Key, X-Amz-Security-Token, locale",
           "Access-Control-Allow-Methods": "GET, POST",
+          Authorization: "Basic " + document.cookie.split("auth=")[1].split(";")[0],
         },
         body: formData,
       })
     ).json();
-    return data;
+
+    const auth = document.cookie.split("auth=")[1].split(";")[0];
+    return await (await fetch(api_url + "/inspections", {
+      method: "POST",
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Allow-Credentials": "true",
+        "Access-Control-Allow-Headers":
+          "Origin, Content-Type, X-Amz-Date, Authorization, X-Api-Key, X-Amz-S",
+        "Content-Type": "application/json",
+        Authorization: "Basic " + auth
+      },
+      body: JSON.stringify({ data: data }),
```

Comment:
The backend expects the bare data, not encapsulated in another object "data".
I suggest: `body: JSON.stringify(data),`